packages/backend/src/peripherals/database/AggregatedReportRepository.ts

```diff
@@ -112,23 +112,33 @@ export class AggregatedReportRepository extends BaseRepository {
     return row ? toRecord(row) : undefined
   }
 
-  async addOrUpdateMany(reports: AggregatedReportRecord[]) {
+  async addOrUpdateMany(
+    reports: AggregatedReportRecord[],
+    where: { timestamp: UnixTime },
+  ) {
+    if (reports.length === 0) {
+      return 0
```

Comment:
we still need to remove records from the database in this case

Reply:
https://github.com/l2beat/l2beat/pull/1838/commits/65b872dba0346c18c980447d4e516c3ebbd9f1ed